### PR TITLE
refactored some stuff

### PR DESCRIPTION
**Pull request by jans-code**
Heya,
I got the kernel ready to be installed via pip and added a logo file.
Thanks for creating this kernel.
G'day!

### CODE REVIEW

**Comment by vickumar1981** [comment]
@jans-code   this is awesome.  thanks!  will take a look and try to merge it in by this weekend.

**Comment by vickumar1981** [comment]
lgtm.

@jans-code   thanks for this!!

File: jupyterdartkernel/kernel.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{"argv":["python","-m","jupyter-dart-kernel", "-f", "{connection_file}"],
```
Comment (vickumar1981) [inline jupyterdartkernel/kernel.json:1]: shouldn't this actually be "jupyterdartkernel" b/c that's the new name of the python module?

i merged it and i think i'm getting an error w/ this line?

`No module named jupyter-dart-kernel`

@jans-code does changing this file, fix that error?

thanks.

Comment (jans-code) [reply jupyterdartkernel/kernel.json:1]: yes that fixes the error :)

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "jupyterdartkernel"
+version = "0.0.1"
+authors = [
+  { name="Vic Kumar", email="vickumar@gmail.com" },
```
Comment (vickumar1981) [inline pyproject.toml:9]: @jans-code  feel free to add yourself to the list of authors as well.

**Pull request merged**